normandy/recipes/models.py

```diff
@@ -36,33 +70,41 @@ class Recipe(models.Model):
 
     # Fields that determine who this recipe is sent to.
     enabled = models.BooleanField(default=False)
-    locale = models.ForeignKey(Locale, blank=True, null=True)
-    country = CountryField(blank=True, null=True, default=None)
+    locales = models.ManyToManyField(Locale, blank=True)
+    countries = models.ManyToManyField(Country, blank=True)
     start_time = models.DateTimeField(blank=True, null=True, default=None)
     end_time = models.DateTimeField(blank=True, null=True, default=None)
     sample_rate = PercentField(default=100)
+    release_channels = models.ManyToManyField(ReleaseChannel, blank=True)
 
     def log_rejection(self, msg):
         logger.debug('{} rejected: {}'.format(self, msg))
 
+    def check_many(self, name, field_qs, client_val):
```

Comment:
This should be named `matches_something`. With something being a better word that I cannot come up with currently. :P

Also a docstring would be nice.